### PR TITLE
Fix broken link in PyTorch loss functions documentation

### LINKED ISSUE #5735: [Edit] Loss Functions - neural network broken link
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting

### Description (optional)

`docs/content/pytorch/concepts/nn/terms/loss-functions/loss-functions.md`

The link for neural network isn't formatted correctly in the introduction. The pointer changes, but there isn't a link attached.

`[neural network's](<(https://www.codecademy.com/resources/docs/pytorch/nn)>)`
should be
`[neural network's](https://www.codecademy.com/resources/docs/pytorch/nn)`

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Maverick073**
### Issue Solved
Closes #5735 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mamtawardhani — COMMENTED** [review]
Thank you for contributing to Codecademy Docs @Maverick073  😄 

The entry looks good for a second round of review! 🚀

**Review by PragatiVerma18 — APPROVED** [review]
Hey @Maverick073, this looks good to go! 🚀

**Comment by Maverick073** [comment]
Thank you @PragatiVerma18 , looking forward to contribute more to the community 😄
